### ISSUE TITLE
Attempt to re-add render abandoned/cancelled marks

### ISSUE THREAD

**Issue by taneliang** [MLH-Fellowship/react#30]
Following discussions with Brian at https://github.com/MLH-Fellowship/react/pull/11#discussion_r442308499, we've removed `markRenderAbandoned` and its `--render-cancel` user timing mark as it could have false positives in certain situations. Brian suggests that we bring this up again once the main React PR is merged into React's main repo.
